package.json

```diff
@@ -256,6 +276,21 @@
                 "command": "aws.showErrorDetails",
                 "title": "%AWS.command.showErrorDetails%",
                 "category": "AWS"
+            },
+            {
+                "command": "aws.viewLogs",
+                "title": "%AWS.command.viewLogs%",
+                "category": "AWS"
+            },
+            {
+                "command": "aws.help",
+                "title": "%AWS.command.help%",
+                "category": "AWS"
+            },
+            {
+                "command": "aws.github",
```

Comment:
Nit: Command IDs should be actions, i.e. `command.viewSource` or `command.viewGitHub`.

Reply:
Looks like that's the case for the string names. We don't have any commands following the `aws.command.*` pattern for the commands themselves. These are inline with the other commands.

Reply:
👍